pandas/tests/indexes/test_range.py

```diff
@@ -1037,3 +1045,19 @@ def test_engineless_lookup(self):
             assert idx.get_loc("a") == -1
 
         assert "_engine" in idx._cache
+
+    def test_reindex_limit(self):
+        # GH 28631
```

Comment:
this doesn’t belong here
rather with other dataframe reindex tests

look in tests/frame

Reply:
okay, move to `tests/frame/test_indexing`